### ISSUE TITLE
Wrong cursor on hover for TextInput multiline={false} editable={false}

### ISSUE THREAD

**Issue by jbcullis** [microsoft/react-native-macos#926]
### Environment

```markdown
react-native 0.64.0:
npm ls react-native-macos: 0.64.13
node: 17.0.1
npm: 8.1.0
yarn: N/A
xcodebuild: 13A1030d
```

### Steps to reproduce the bug

Create the following text input:
<TextInput multiline={true} editable={false} />

Hover over and the cursor stays as the text selection cursor.

### Expected Behavior

When editable is false, mouse should revert to default cursor.

### Actual Behavior

Cursor remains as text selection cursor.

**Comment by HeyImChris** [comment]
@jbcullis Would you mind linking a sample project we can repro this with?

**Comment by jbcullis** [comment]
Will do @HeyImChris - on the road this week but will get a sample up this weekend.

**Comment by harrieshin** [comment]
@mischreiber / @Saadnajmi : https://github.com/microsoft/react-native-macos/pull/760 does this address this issue?

**Comment by Saadnajmi** [comment]
I'm not sure. I think there's a followup change after #760 and #1346 needed to set the default cursors. @Shwanton could you confirm?

**Comment by shwanton** [comment]
Now that #1346 is merged, we will add cursor customization for NSTextView like #760

**Comment by Saadnajmi** [comment]
Sorry @Shwanton I thought this issue was about Text, it's actually about TextInput, so I don't think your change is applicable. I looked at our last few commits and I didn't see any obvious change that fixes this particular issue. @jbcullis could you confirm you still have this on the latest `main` commit?

**Comment by shwanton** [comment]
Ah, my bad! I didn't realize this was for TextInput.

**Comment by jbcullis** [comment]
@Saadnajmi will do, just need a couple days to get the project on latest.

**Comment by jbcullis** [comment]
Resolved in 68.5
